code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -413,13 +469,17 @@
 
 /datum/reagent/medicine/pen_acid/on_mob_life(mob/living/M)
 	if(M.radiation > 0)
-		M.radiation -= 4
-	M.adjustToxLoss(-2*REM, 0)
+		M.radiation -= 7
+	if(prob(75))
+		M.adjustToxLoss(-4*REM, 0)
```

Comment:
Why not just use M.adjustToxLoss(-3*REM, 0) instead of the if statement? Using probabilities are helpful to ensure health is a binary fraction, but that's not applicable here.